torch/csrc/autograd/VariableTypeUtils.h

```diff
@@ -148,7 +148,7 @@ inline std::vector<SavedVariable> make_saved_variable_list(TensorList tensors) {
 }
 
 inline Tensor as_variable(Tensor tensor) {
-  return make_variable(std::move(tensor), /*requires_grad=*/false);
+  return make_variable(tensor, /*requires_grad=*/false);
```

Comment:
If the original code is doing `std::move(tensor)` already, why do we need to change it back to the slow path?

Reply:
@yf225 `std::move` was a no-op here before I added the rvalue reference overload of `make_variable`, so I am retaining the old behavior. I didn't want to mess with stuff unrelated to what I was changing.